addons/website_event/static/src/scss/event_templates_list.scss

```diff
@@ -128,4 +128,8 @@
             max-width: 300px;
         }
     }
+    .o_wevent_filter_count_pill {
```

Comment:
The class "badge" seems more appropriate for this than fixed dimensions, and handles the case of bigger numbers (even though I agree it should not happen but hey).

![image](https://github.com/user-attachments/assets/5a805427-b9ec-40ff-9d46-ecd5c36c4df6)